packages/cli/src/lib/config.ts

```diff
@@ -38,6 +39,14 @@ async function createConfig(): Promise<Config> {
 }
 
 export async function loadOrCreateConfig(path = DEFAULT_CONFIG_PATH): Promise<Config> {
+  if (!existsSync(path)) {
+    // no new named config file found
+    if (existsSync(join(homedir(), '/.llama-pack/llama-pack-config.json'))) {
+      // old named config file found - rename it and it's folder
```

Comment:
nit: It should be "its", not "it's" (I only know this by getting it wrong so many times).